src/twisted/conch/ssh/transport.py

```diff
@@ -92,21 +102,28 @@ class SSHCiphers:
     @ivar inMAc: see outMAC, but for the incoming MAC.
     """
 
+    _deprecatedCipherMap = {
+        b"cast128-cbc": (algorithms.CAST5, 16, modes.CBC),
+        b"cast128-ctr": (algorithms.CAST5, 16, modes.CTR),
+        b"blowfish-ctr": (algorithms.Blowfish, 16, modes.CTR),
+        b"blowfish-cbc": (algorithms.Blowfish, 16, modes.CBC),
+    }
     cipherMap = {
+        # 3DES, ARC4 are also legacy (and ARC4 has serious security issues),
```

Comment:
Why should we care here about ARC4 ?

And these days many automated SSH scanner will complain about any CBC mode...

I am not sure that we should add comment about these things here.

My suggestion is to just remove this comment.